learn/core_concepts/indexes.md

```diff
@@ -18,7 +18,11 @@ One index's settings do not impact other indexes. For example, you could use a d
 
 ## Index creation
 
-If you try to add documents or settings to an index that does not already exist, Meilisearch will automatically create it for you. You can also create indexes manually using the [create index endpoint](/reference/api/indexes.md#create-an-index).
+If you try to add documents or settings to an index that does not already exist, Meilisearch will automatically create it for you.
```

Comment:
Using this phrase makes it a bit easier to describe the differences between the two options.

```suggestion
### Implicit index creation

If you try to add documents or settings to an index that does not already exist, Meilisearch will automatically create it for you.
```